west.yml

```diff
@@ -139,6 +139,13 @@ manifest:
       path: modules/hal/ti
       groups:
         - hal
+    - name: hal_we
+      revision: bb79ef1cc46d00058e89b48ad36cca8efebbbf0e
+      # Explicit URL (temporary)
```

Comment:
temporary?

Reply:
Yes - the "hal_we" repository is currently hosted outside the Zephyr project, until the new module repository has been created.

See https://github.com/zephyrproject-rtos/zephyr/issues/47021#issuecomment-1190499840

Reply:
The new module repo has been created

Reply:
I don't have access to the repo yet - so I made all the requested changes except for this one.